File: Bungii.Test.Integration/src/test/java/com/bungii/web/pages/driver/Driver_VerifyPhonePage.java
package com.bungii.web.pages.driver;

import com.bungii.common.core.PageBase;
import org.openqa.selenium.WebElement;

public class Driver_VerifyPhonePage extends PageBase {

    //---------------------------Forgot Password - Verify Your Phone-------------------------------------------------
    //Driver - Verify your phone - Header
    public WebElement Header_VerifyPhone () { return findElement("//div[@id='verification']/h2", LocatorType.XPath); }

    //Driver - Verify your phone - text
    public WebElement Text_Verify_PhoneNo () { return findElement("//div[@id='verification']/p", LocatorType.XPath); }

    //Verify Your Phone - Blank field validation error
    public WebElement Err_VerifyPhone_BlankField () { return findElement("verificationerrorsummary", LocatorType.Id); }

    //Verify Your Phone - Resend Code
    public WebElement Button_ResendCode () { return findElement("btnResendSMS", LocatorType.Id); }

    //Verify Your Phone - Code
    public WebElement Textfield_Code () { return findElement("SMSVerificationCode", LocatorType.Id); }

    //Verify Your Phone - Password
    public WebElement Textfield_Password () { return findElement("new-password", LocatorType.Id); }

    //Verify Your Phone - Confirm Password
    public WebElement Textfield_ConfirmPassword () { return findElement("confirm-password", LocatorType.Id); }

    //Verify Your Phone - Password - Passwords dont match -error
    public WebElement Button_ResetPassword () { return findElement("//form[@id='ResetPassword']/button[contains(text(),'Reset')]", LocatorType.XPath); }

    //Verify Your Phone - Blank field validation error
    public WebElement Err_VerifyPhone_BlankPasswords () { return findElement("verificationerrorsummary", LocatorType.Id); }

    //Verify Your Phone - Incorrect Verification Code
    public WebElement Err_VerifyPhone_Code_Incorrect () { return findElement("//form[@id='ResetPassword']/p/span", LocatorType.XPath); }

    //Verify Your Phone - Password - error
    public WebElement Err_VerifyPhone_Password_Invalid () { return findElement("new-password-error", LocatorType.Id); }

    //Verify Your Phone - Password - Passwords dont match -error
    public WebElement Err_VerifyPhone_ConfirmPassword () { return findElement("confirm-password-error", LocatorType.Id); }
}

File: BungiiAutomation/Bungii.Test.Regression.Web.Integration/Pages/Driver/Driver_VerifyPhonePage.cs
using OpenQA.Selenium;
using OpenQA.Selenium.Support.PageObjects;

namespace Bungii.Android.Regression.Test.Integration.Pages.Driver
{
    public class Driver_VerifyPhonePage
    {
        public Driver_VerifyPhonePage(IWebDriver webdriver)
        {
            PageFactory.InitElements(webdriver, this);
        }
        //---------------------------Forgot Password - Verify Your Phone-------------------------------------------------
        //Driver - Verify your phone - Header
        [FindsBy(How = How.XPath, Using = "//div[@id='verification']/h2")]
        public IWebElement Header_VerifyPhone { get; set; }

        //Driver - Verify your phone - text
        [FindsBy(How = How.XPath, Using = "//div[@id='verification']/p")]
        public IWebElement Text_Verify_PhoneNo { get; set; }

        //Verify Your Phone - Blank field validation error
        [FindsBy(How = How.Id, Using = "verificationerrorsummary")]
        public IWebElement Err_VerifyPhone_BlankField { get; set; }

        //Verify Your Phone - Resend Code
        [FindsBy(How = How.Id, Using = "btnResendSMS")]
        public IWebElement Button_ResendCode { get; set; }

        //Verify Your Phone - Code
        [FindsBy(How = How.Id, Using = "SMSVerificationCode")]
        public IWebElement Textfield_Code { get; set; }

        //Verify Your Phone - Password
        [FindsBy(How = How.Id, Using = "new-password")]
        public IWebElement Textfield_Password { get; set; }

        //Verify Your Phone - Confirm Password
        [FindsBy(How = How.Id, Using = "confirm-password")]
        public IWebElement Textfield_ConfirmPassword { get; set; }

        //Verify Your Phone - Password - Passwords dont match -error
        [FindsBy(How = How.XPath, Using = "//form[@id='ResetPassword']/button[contains(text(),'Reset')]")]
        public IWebElement Button_ResetPassword { get; set; }

        //Verify Your Phone - Blank field validation error
        [FindsBy(How = How.Id, Using = "verificationerrorsummary")]
        public IWebElement Err_VerifyPhone_BlankPasswords { get; set; }

        //Verify Your Phone - Incorrect Verification Code
        [FindsBy(How = How.XPath, Using = "//form[@id='ResetPassword']/p/span")]
        public IWebElement Err_VerifyPhone_Code_Incorrect { get; set; }

        //Verify Your Phone - Password - error
        [FindsBy(How = How.Id, Using = "new-password-error")]
        public IWebElement Err_VerifyPhone_Password_Invalid { get; set; }

        //Verify Your Phone - Password - Passwords dont match -error
        [FindsBy(How = How.Id, Using = "confirm-password-error")]
        public IWebElement Err_VerifyPhone_ConfirmPassword { get; set; }
    }
}
